chia/rpc/full_node_rpc_api.py

```diff
@@ -397,6 +405,48 @@ async def get_block_records(self, request: Dict) -> Optional[Dict]:
             records.append(record)
         return {"block_records": records}
 
+    async def get_block_spends(self, request: Dict) -> Optional[Dict]:
+        if "header_hash" not in request:
+            raise ValueError("No header_hash in request")
+        header_hash = bytes32.from_hexstr(request["header_hash"])
+        full_block: Optional[FullBlock] = await self.service.block_store.get_full_block(header_hash)
+        if full_block is None:
+            raise ValueError(f"Block {header_hash.hex()} not found")
+
+        tx_info: Optional[TransactionsInfo] = full_block.transactions_info
+        generator_program: Optional[SerializedProgram] = full_block.transactions_generator
+        height: uint32 = full_block.reward_chain_block.height
+        spends: List[CoinSpend] = []
+        if tx_info is not None and generator_program is not None:
+            cost = tx_info.cost
+            block_generator = await self.service.blockchain.get_block_generator(full_block)
+            if block_generator is None:
+                return {"block_spends": spends}
+
+            if height >= DEFAULT_CONSTANTS.SOFT_FORK_HEIGHT:
+                # conditions must use integers in canonical encoding (i.e. no redundant
+                # leading zeros)
+                # the division operator may not be used with negative operands
+                flags = COND_CANON_INTS | NO_NEG_DIV
+            else:
+                flags = 0
```

Comment:
technically, since this doesn't do any validation, you can just assume that the block you find satisfies the consensus rules. So, I don't think you need to specify these flags.